NiceIO.Tests/ChangeExtension.cs

```diff
@@ -40,5 +40,17 @@ public void ToEmptyString()
 			var actual = new NPath("/my/path/file.something.exe").ChangeExtension(string.Empty);
 			Assert.AreEqual(expected, actual);
 		}
+
+		[Test]
+		public void BlockedOnWindowsRoot()
```

Comment:
Yes. I'll push the improved names straight to master.  a PR feels like over kill.